content/en/docs/components/model-registry/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+  - tarilabs
+  - rareddy
+  - ederign
+  - Tomcli
+  - andreyvelich
```

Comment:
You don't need to keep me and @zijianjoy in the model-registry Owners.
Just keep the active contributors from Model Registry side.
Also, please try to keep this list alphabetically.

Reply:
done with 8e8a075e0aa38f9ead871516fcfad4a15f9c369b 👍